src/Validator/WhiteList.php

```diff
@@ -75,6 +75,30 @@ public function getDescription()
         return 'Value must be one of (' . \implode(', ', $this->list) . ')';
     }
 
+    /**
+     * Is array
+     *
+     * Function will return true if object is array.
+     *
+     * @return bool
+     */
+    public function isArray(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get Type
+     *
+     * Returns validator type.
+     *
+     * @return string
+     */
+    public function getType()
+    {
+        return self::TYPE_ARRAY;
```

Comment:
@kodumbeats  @eldadfux  can this be replaced with `$this->validator->getType()`

Reply:
As eldad, mentioned let's also explore the possibility of replacing it with an Enum.

Reply:
We can't use `$this->validator->getType()` because this validator just accepts a list of possible values and not a validator. Maybe we can accept the type as a new param in the constructor? (default: string?) or just use a function to detect the first option type?

Reply:
I think it's appropriate to have whitelists default to string and be declarative if it deviates (to avoid any unexpected behavior). Most of the whitelist instances in `appwrite/appwrite` are strings.